.github/workflows/git-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Create new release for OpenWIFI 2.x services
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release version to create (i.e. v2.4.0)'
+        required: true
+      tag_type:
+        default: 'rc'
+        description: 'Tag type to create (final OR rc)'
+        required: true
+      push_confirm:
+        default: ''
+        description: 'Set to "true" if changes should be pushed (dry-run enabled by default)'
+        required: false
+      log_level:
+        default: 'INFO'
+        description: 'Log level for the script'
+        required: false
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+      with:
+        path: wlan-cloud-ucentral-deploy
+
+    - name: Install system dependencies
+      run: |
+        pip3 install yq
+        helm plugin install https://github.com/databus23/helm-diff
+        helm plugin install https://github.com/aslafy-z/helm-git --version 0.10.0
+
+    - name: Run release creation script
+      working-directory: wlan-cloud-ucentral-deploy/.github/git-release-tool
+      run: |
+        export RELEASE_VERSION=${{ github.event.inputs.release_version }}
+        export TAG_TYPE=${{ github.event.inputs.tag_type }}
+        export GIT_PUSH_CONFIRMED=${{ github.event.inputs.push_confirm }}
+        export LOG_VERBOSITY=${{ github.event.inputs.log_level }}
+        git config --global credential.helper store
+        git config --global user.email "dmitry@opsfleet.com"
+        git config --global user.name "Dmitry Dunaev"
```

Comment:
I think we should switch to the tip-automation-user here so it's evident that the changes were made by an automated process.

Reply:
Do we have one that we are using at the moment?

Reply:
I guess https://github.com/tip-automation could be used for this case.

Reply:
Updated